src/status_im/utils/http.cljs

```diff
@@ -2,13 +2,38 @@
   (:require [status-im.utils.utils :as utils]
             [status-im.react-native.js-dependencies :as rn-dependencies]
             [taoensso.timbre :as log]
-            [goog.Uri :as uri]
             [clojure.string :as string])
   (:refer-clojure :exclude [get]))
 
 ;; Default HTTP request timeout ms
 (def http-request-default-timeout-ms 3000)
 
+(defn- headers [response]
+  (let [entries (es6-iterator-seq (.entries (.-headers response)))]
+    (reduce #(assoc %1 (string/trim (string/lower-case (first %2))) (string/trim (second %2))) {} entries)))
```

Comment:
it looks a bit messy and can be simplified, but seems like it's not really related to this pull request